server/base/src/main/java/org/apache/accumulo/server/tabletserver/MemoryManager.java

```diff
@@ -32,8 +33,22 @@
 
   /**
    * Initialize the memory manager.
+   *
+   * @deprecated since 2.1 use {@link #init(ServerContext)}
```

Comment:
I'd add the note about scheduled for removal in this javadoc tag also.
```suggestion
   * @deprecated since 2.1 use {@link #init(ServerContext)}; to be removed in 3.0.0
```